clang/test/Driver/compress.c

```diff
@@ -1,5 +1,9 @@
+// XFAIL: system-aix
+
 // REQUIRES: zlib
 
+// XFAIL: target={{.*}}-aix{{.*}}
```

Comment:
system-aix is sufficient and `// XFAIL: target={{.*}}-aix{{.*}}` is uneeded?

Reply:
That seems correct, I've removed `// XFAIL: target={{.*}}-aix{{.*}}`